AutoRest/Generators/Extensions/Extensions/Extensions.cs

```diff
@@ -410,13 +410,18 @@ public static void AddParameterGroups(ServiceClient serviceClient)
                         parameterGroupType = new CompositeType
                         {
                             Name = parameterGroupName,
-                            Documentation = "Additional parameters for the " + method.Name + " operation."
+                            Documentation = "Additional parameters for these operations: " + method.Name
```

Comment:
The documentation does wrap in the summary and intellisense truncates the text but we certainly don't need to list them.  How about just a simple summary of "Additional parameters for one or more operations" ?